uber/templates/base.html

```diff
@@ -69,7 +69,8 @@
             {% endif %}
             {% if c.HAS_STUFF_ACCESS %}
                 {Schedule: [
-                    {'View Schedule': '../schedule/'},
+                    {% if c.ALT_SCHEDULE_URL %}{'View External Schedule': '../schedule/'},{% endif %}
```

Comment:
This is almost perfect, but unfortunately `c.ALT_SCHEDULE_URL` is only a variable that exists in the panels plugin, so we can't use it here. I'm not sure off the top of my head the best way to display this for admins, maybe just make this link "View Schedule" and the next link "View Schedule (Internal Only)" and that way it may not be terribly confusing if both link to the same thing?